apps/design-system/package.json

```diff
@@ -20,7 +21,9 @@
     "generate:assets": "echo \"No assets to configure\"",
     "test": "echo \"No tests\"",
     "eject": "react-scripts eject",
-    "dev": "storybook dev -p 6006 --no-open",
+    "dev": "concurrently 'npm run watch:*'",
+    "watch:storybook": "storybook dev -p 6006 --no-open",
+    "watch:tailwind": "npx tailwindcss -i ./src/styles/tailwind.css -o ./src/styles/tailwind.output.css --watch",
```

Comment:
because we need a way to style the form in design system. and in studio we will be using tailwind to style the form, that's why I am using tailwind to put together the form elements here.